<launch>
  <node pkg="hkvs_cam" name="hkvs_cam_type" type="hkvs_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="camera_info_url" value="package://hkvs_cam/camera_info/head_camera.yaml"/>
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
</launch>
